tests/testsuite.src/syn_definition.at

```diff
@@ -2878,3 +2878,76 @@ prog.cob:6: error: 'DEPTHDR' is not defined
 
 AT_CLEANUP
 
+
+# Note: the three tests below should be combined in a single one (when fully supported)
```

Comment:
hm. let's drop the comment and rather extend the checks later (using the pointers)